conf/evolutions/reversions/003-add-organization-displayname.sql

```diff
@@ -0,0 +1,5 @@
+START TRANSACTION;
```

Comment:
Looks like I just forgot to change the name of the evolution, so we should remove one of the files

Reply:
agree. @jstriebel should the CI check have caught this?

Reply:
What exactly? The CI only checks the forward-evolutions, to prevent using the same number with merges. See https://github.com/scalableminds/webknossos/blob/master/tools/postgres/assert_unique_evolution_numbers.sh

Reply:
I see, thanks